### PR TITLE
Newsletters: add 278 (2023-11-22)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, update releases/RCs, add topic links @harding 
- [x] Client/Services section @bitschmidty 

Note: we're planning to do the spaces recap on Wednesday (22 Nov) instead of the usual Thursday due to U.S. thanksgiving.  (Maybe we want to publish the newsletter on Tuesday?  I'm ok with that.)

### CODE REVIEW

**Comment by bitschmidty** [comment]
Id prefer the Weds publication consistency over the additional time between news and pod personally.

**Review by bitschmidty — COMMENTED** [review]
lgtm so far

File: _posts/en/newsletters/2023-11-22-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+- [Bitcoin Core 26.0rc2][] is a release candidate for the next major
+  version of the predominant full node implementation. There's a [testing
+  guide][26.0 testing] and a scheduled
+  meeting of the [Bitcoin Core PR Review Club][] dedicated to testing on
+  15 November 2023.
```
Comment (vostrnad) [inline _posts/en/newsletters/2023-11-22-newsletter.md:86]: The testing meeting has already happened:
```suggestion
- [Bitcoin Core 26.0rc2][] is a release candidate for the next major
  version of the predominant full node implementation. A [testing
  guide][26.0 testing] is available.
```

File: _posts/en/newsletters/2023-11-22-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  with the [c-lightning-rest][] plugin.
+
+- [Eclair #2752][] allows data in an [offer][topic offers] to reference
+  a node using either its public key or the identity of one of its
+  channels.  A public key is the typical way to identify a node, but it
+  uses 33 bytes.  A channel can be identified using a [BOLT7][] _short
+  channel identifier_ (SCID), which only uses 8 bytes. Because channels
+  are shared by two nodes, an additional bit is prepended to the SCID to
+  specifically identify one of the two nodes.  Because offers may often
+  be used in size-constrained media, this space savings is significant.
```
Comment (vostrnad) [inline _posts/en/newsletters/2023-11-22-newsletter.md:112]:
```suggestion
  be used in size-constrained media, the space savings are significant.
```

**Review by t-bast — COMMENTED** [review]
LGTM for the `Offers-compatible LN addresses` section, thanks for this nice summary!

File: _posts/en/newsletters/2023-11-22-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+- **Athena Bitcoin adds Lightning support:**
+  The Bitcoin ATM [operator][athena website] recently [announced][athena tweet]
+  support for receiving Lightning payments for cash withdrawals.
+
+- **Blixt v0.6.9 released:**
+  The [v0.6.9][blixt v0.6.9] release includes support for simple taproot
+  channels, defaults to [bech32m][topic bech32] receive addresses, and adds
+  additional [zero conf channel][topic zero-conf channels] support.
+
+- **Durabit whitepaper announced:**
```
Comment (harding) [inline _posts/en/newsletters/2023-11-22-newsletter.md:90]: Regarding the two papers in this section, I think we should keep them for this newsletter, but I would prefer in the future that we encourage the authors of ideas to post them to one of our [sources](https://bitcoinops.org/internal/sources/).  We want ideas like these to have a chance to be peer reviewed before we write about them.

File: _posts/en/newsletters/2023-11-22-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+  context, a popular [lightning address][] standard exists based on
+  [LNURL][], which requires operating an always-available HTTP server to
+  associate email-style addresses with LN invoices.  Teinturier notes
+  that this creates several problems:
+
+    - _Lack of privacy:_ the server operator likely learns the IP
+      address of the spender and the receiver.
+
+    - _Risk of theft:_ the server operator can man-in-the-middle
+      invoices in order to steal funds.
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-11-22-newsletter.md:31]:
```suggestion
      invoices to steal funds.
```

File: _posts/en/newsletters/2023-11-22-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+      (according to a certificate authority) the owner of example.com
+      claimed this particular node is controlled by alice@example.com.
+      Teinturier notes that this would require LN implementations to
+      implement SSL-compatible cryptography.
+
+    - _Store offers directly in DNS:_ a domain may have multiple DNS
+      records that directly store offers for particular addresses.  For
+      example, a DNS `TXT` record, `alice._lnaddress.domain.com`,
+      includes an offer for Alice.  Another record,
+      `bob._lnaddress.domain.com`includes an offer for Bob.  Teinturier
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-11-22-newsletter.md:63]:
```suggestion
      `bob._lnaddress.domain.com` includes an offer for Bob.  Teinturier
```

File: _posts/en/newsletters/2023-11-22-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+      notes that this requires the domain owner create one DNS record
+      per user (and update that record if the user needs to change their
+      default offer).
+
+  As of this writing, there had not been any public discussion of the
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-11-22-newsletter.md:68]:
```suggestion
  As of this writing, there has not been any public discussion of the
```

File: _posts/en/newsletters/2023-11-22-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+- **Durabit whitepaper announced:**
+  The [Durabit whitepaper][] outlines a protocol using [timelocked][topic
+  timelocks] Bitcoin transactions in conjunction with a chaumian-style mint
+  to incentivize seeding of large files.
+
+- **BitStream whitepaper announced:**
+  The [BitStream whitepaper][] and [early prototype][bitstream github] layout a
+  protocol for the hosting and atomic exchange of digital content for
+  coins using timelocks and merkle trees with verification and fraud
+  proofs.  For previous discusison of paid data transfer protocols, see
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-11-22-newsletter.md:103]:
```suggestion
  proofs.  For previous discussion of paid data transfer protocols, see
```

File: _posts/en/newsletters/2023-11-22-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+
+- **Blixt v0.6.9 released:**
+  The [v0.6.9][blixt v0.6.9] release includes support for simple taproot
+  channels, defaults to [bech32m][topic bech32] receive addresses, and adds
+  additional [zero conf channel][topic zero-conf channels] support.
+
+- **Durabit whitepaper announced:**
+  The [Durabit whitepaper][] outlines a protocol using [timelocked][topic
+  timelocks] Bitcoin transactions in conjunction with a chaumian-style mint
+  to incentivize seeding of large files.
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-11-22-newsletter.md:97]:
```suggestion
  to incentivize the seeding of large files.
```

File: _posts/en/newsletters/2023-11-22-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.17.2-beta][] is a maintenance release that only includes one
+  small change to fix the bug reported in [LND #8186][].
+
+- [Bitcoin Core 26.0rc2][] is a release candidate for the next major
+  version of the predominant full node implementation. There's a [testing
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-11-22-newsletter.md:126]:
```suggestion
  version of the predominant full-node implementation. There's a [testing
```

**Comment by harding** [comment]
Made edits for all feedback (thanks @vostrnad @bitschmidty @LarryRuane @t-bast !), added lede, updated releases/RCs, and added topic entries.  Reviewed client/services section (thanks @bitschmidty !)  Thanks everyone!

File: _posts/en/newsletters/2023-11-22-newsletter.md (hunk 3 above)
Comment (bitschmidty) [reply _posts/en/newsletters/2023-11-22-newsletter.md:96]: I think this is fair

**Comment by bitschmidty** [comment]
Reviewed topic entries, lede, and latest review comments. Squashed and merged, thank you all!

File: _posts/en/newsletters/2023-11-22-newsletter.md (hunk 9 above)
Comment (vostrnad) [reply _posts/en/newsletters/2023-11-22-newsletter.md]: This doesn't look right. A grep through the `_posts/en` directory gives 240 results for "full node" and only 6 for "full-node", all of which are a part of a URL.

Comment (harding) [reply _posts/en/newsletters/2023-11-22-newsletter.md]: I don't have a strong preference either way.  I suggest we leave it as-is in the merged newsletter and I'll try to remember this discussion the next time someone suggests the hyphenated / single-adjective version and we can debate it then.

**Pull request merged**